src/org.xtuml.bp.doc/Reference/OAL/Relationships/Relationships.adoc

```diff
@@ -0,0 +1,340 @@
+
+Relationship navigation is the function whereby relationships specified on 
+the Class Diagram are read in order to determine the instance or set of 
+instances that are related to an instance of interest.
+
+=== Syntax
+
+----
+select one <instance handle> related by <start> -> <relationship link> -> ... <relationship link>;
+
+select any <instance handle> related by <start> -> <relationship link> -> ... <relationship link>;
+
+select many <instance handle set> related by <start> -> <relationship link> -> ... <relationship link>;
+
+select one <instance handle> related by <start> -> <relationship link> -> ... <relationship link> where <where expression>;
+
+select any <instance handle> related by <start> -> <relationship link> -> ... <relationship link> where <where expression>;
+
+select many <instance handle set> related by <start> -> <relationship link> -> ... <relationship link> where <where expression>;
+----
+
+`+<start>+` is an `+<instance handle set>+` or `+<instance handle>+` obtained from a previous select statement.
+
+`+<relationship link>+` is a `+<keyletter>[<relationship specification>]+` , where the square brackets are literal 
+and do not indicate optional text.
+
+`+<keyletter>+` is the keyletter of the class reached by the specified relationship.
+
+`+<relationship specification>+` is the specification of the relationship from the source to the destination class.
+
+`+<where expression>+` is a type of boolean expression using the `+selected+` keyword.
+
+=== Notes
+
+A relationship link chain is the sequence of `+<relationship link>+` 's used to specify the path 
+from the starting instance or set of instances to the destination.
+
+Use the `+select one+` form if at most one instance handle can be returned by navigating 
+the relationship link chain.
+
+Use the `+select any+` or `+select many+` form if more than one instance handle 
+can be returned by navigating the relationship link chain. `+select any+` returns 
+a single instance, and `+select many+` returns all instances that meet the 
+selection criteria.
+
+The `+select any+` form returns the instance handle of an arbitrary instance of 
+the class at the end of the relationship link chain.
+
+The `+select many+` form returns an instance handle set containing all the 
+instances of the class at the end of the relationship link chain.
+
+The `+select any ... where+` form returns the instance handle of an arbitrary 
+instance of the class at the end of the relationship link chain that fulfills 
+the `+<where expression>+` criteria.
+
+The `+select many ... where+` form returns an instance handle set containing 
+all the instances of the class at the end of the instance chain that fulfill 
+the `+<where expression>+` criteria.
+
+The relationship phrases in the relationship link chain must be given in the 
+direction of navigation.
+
+If the starting `+<instance handle>+` or `+<instance handle set>+` is empty, 
+then the result will be considered a run time error.
```

Comment:
This is not true. If the starting instance handle or instance handle set is empty then the resulting instance handle or instance handle set will be guaranteed to be empty, but it is still valid.

Reply:
Good catch.  I resisted the urge to revise the text in the entire section, but I fixed this one.

Reply:
Ah, makes sense. I totally understand that "urge" when updating the old docs.